src/Tribe/Template_Factory.php

```diff
@@ -538,6 +469,86 @@ public function comments_off( $option_value, $option_name ) {
 		}
 
 		return $option_value;
+	}
+
+	/************************
+	 *                      *
+	 *  Deprecated Methods  *
+	 *                      *
+	 ************************/
 
+	/**
+	 * Asset calls for vendor packages
+	 *
+	 * @deprecated TBD
+	 *
+	 * @param string $name
+	 * @param array  $deps Dependents
+	 */
+	public static function asset_package( $name, $deps = array() ) {
+
+		$common = Tribe__Events__Main::instance();
+		$prefix = 'tribe-events'; // Tribe__Events__Main::POSTTYPE;
```

Comment:
The post type is `tribe_events` tho not sure if the comment should be more explicit about that

Reply:
It's deprecated code.

Reply:
does using `_deprecated_function()` make sense here and for these other deprecated asset methods?

Reply:
Nopes, other plugins will use these methods for now. We want to avoid deprecating it with notices for now.